### DISCUSSION TITLE
问题

### DISCUSSION THREAD

**Discussion by cqray1990** [PaddlePaddle/PaddleOCR#13475] [Q&A]
通用文字识别，多语言的，组成一个包含各种语言的字典去识别的更好些，还是去分语种呢，分语言，还要对文本进行分类，流程较多

**Comment by mattheliu** [comment]
你好，多语言识别的字典应该分语种分语言

**Comment by UserWangZz** [comment]
字典是按语种分的，并且由于历史原因，字典的命名是按照主要的语言命名的
